src/style.css

```
@@ -0,0 +1,130 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+}
+
+.card__photo {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__info {
+  padding-bottom: 16px;
+}
+
+.card__text-name {
+  font-weight: 500;
+  margin: 0 0 4px;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__product-code {
+  font-size: 10px;
+  color: #616070;
+}
+
+.card__stars-text {
+  margin-bottom: 24px;
+  display: flex;
+  flex-direction: row;
+  align-items: flex-end;
+  justify-content: space-between;
+}
+
+.card__stars {
+  display: flex;
+}
+
+.card__item {
+  display: block;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(images/star.svg);
+}
+
+.card__stars--1 .card__item:nth-child(-n + 1) {
+  background-image: url(images/star-active.svg);
+}
+
+.card__stars--2 .card__item:nth-child(-n + 2) {
+  background-image: url(images/star-active.svg);
+}
+
+.card__stars--3 .card__item:nth-child(-n + 3) {
+  background-image: url(images/star-active.svg);
+}
+
+.card__stars--4 .card__item:nth-child(-n + 4) {
+  background-image: url(images/star-active.svg);
+}
+
+.card__stars--5 .card__item:nth-child(-n + 5) {
+  background-image: url(images/star-active.svg);
+}
+
+.card__text-reviews {
+  font-size: 10px;
+  color: #060b35;
+}
+
+.card__price {
+  margin-bottom: 16px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.card__text-price {
+  font-size: 12px;
+  color: #616070;
+}
+
+.card__number-price {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: bold;
```

use numeric value instead of reserved word one
```suggestion
  font-weight: 700;
```